src/app/fdctl/configure/workspace.c

```diff
@@ -329,7 +329,7 @@ init( config_t * const config ) {
         buf    ( pod, "src_mac_addr",                 config->tiles.net.mac_addr, 6 );
         ushort1( pod, "transaction_listen_port",      config->tiles.quic.regular_transaction_listen_port, 0 );
         ushort1( pod, "quic_transaction_listen_port", config->tiles.quic.quic_transaction_listen_port, 0 );
-        ulong1 ( pod, "idle_timeout_ms",              config->tiles.quic.idle_timeout_millis );
+        ulong1 ( pod, "idle_timeout_millis",          1000 );
```

Comment:
This doesn't look right, you want to keep the config value here

Reply:
you're right - I should've paid more attention to the merge. This didn't flag a conflict, weirdly